src/bitmap/inherent.rs

```diff
@@ -578,6 +578,80 @@ impl RoaringBitmap {
 
         None
     }
+
+    /// Removes the specified number of elements from the top.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use roaring::RoaringBitmap;
+    ///
+    /// let mut rb = RoaringBitmap::from_iter([1, 5, 7, 9]);
+    /// rb.remove_first(2);
+    /// assert_eq!(rb, RoaringBitmap::from_iter([7, 9]));
+    /// rb = RoaringBitmap::from_iter([1, 3, 7, 9]);
+    /// rb.remove_first(2);
+    /// assert_eq!(rb, RoaringBitmap::from_iter([7, 9]));
+    pub fn remove_first(&mut self, mut n: usize) {
+        if n > self.len() as usize {
+            return;
+        }
+        // remove containers up to the front of the target
+        let position = self.containers.iter().position(|container| {
+            let container_len = container.len() as usize;
+            if container_len < n {
```

Comment:
This should be `<=`, so we remove the container if it should be fully removed. Same below for remove_last

Reply:
fix
https://github.com/RoaringBitmap/roaring-rs/pull/253/commits/e690f2ac514de2288619cdf5df4012b9c3a81139